file.go

```diff
@@ -51,6 +53,23 @@ func NormalizePath(dir string) (string, error) {
 	return filepath.Clean(dir), nil
 }
 
+// ExpandPath normalises a path and expands it to an absolute form (if it's
+// not already an absolute path).
+func ExpandPath(path string) (string, error) {
+	normPath, err := NormalizePath(path)
+	if err != nil {
+		return "", errors.Annotate(err, "unable to normalise file path")
+	}
+	if !filepath.IsAbs(normPath) {
```

Comment:
Why not just use filepath.Abs(normPath) directly? It's more likely to work correctly on platforms other than Unix.

Also, the docs should probably mention Normalize by name - it's not necessarily clear that "normalising" a path implies expanding the `~` prefix.